HOWTO.md

```diff
@@ -0,0 +1 @@
+# howto
```

Comment:
This line should be uppercased.

Reply:
I think you should put a comma at the end in this line.

Reply:
Thanks, applied the suggested changes.